### ISSUE TITLE
Transformer triggers on both HeartbeatLoop and Loop when LoopTime is equal to the GlobalLoopSecond

### ISSUE THREAD

**Issue by liltinyten** [Eggs-D-Studios/wos-issues#460]
### Guidelines

- [X] I checked for duplicate bug reports
- [X] I tried to find a way to reproduce the bug

### Version

Main (Production)

### What happened? What did you expect to happen?

When a `Transformer`'s `LoopTime` is set to 1. It sends two triggers. When `LoopTime` is set to any value != 1, the bug doesn't happen. Ignore Lolman, the bug still occurs if you reproduce using the steps in the next section.

https://github.com/user-attachments/assets/bda611d9-1364-48d1-b61f-3d65558a08f3

### How can we reproduce this bug?

1. Spawn a power source
2. Spawn a transformer
3. Spawn an anchor or anything that can be triggered.
4. Place the part on the powered transformer
5. Observe it triggering 2 times.

### Frequency

Always

**Comment by hail12pink** [comment]
yeah. the issue in the code is obvious too idk how this could have happened...

**Comment by Hexcede** [comment]
Fixed in next publish
